.gitlab-ci.yml

```diff
@@ -110,9 +110,6 @@ variables:
 .compliance_base: &compliance_base
   stage: 'compliance'
   tags: ['beaker']
-  # There is a bug in inspec that is going to always cause this to fail until it
-  # is fixed: https://github.com/inspec/inspec/pull/4665
```

Comment:
The inspec PR has been merged; we'll see if the GitLab CI pipelines have problems: https://gitlab.com/simp/pupmod-simp-auditd/pipelines/141628069